test/e2e/app-dir/app/flying-shuttle.test.ts

```diff
@@ -73,6 +86,18 @@ import { nextTestSetup, isNextStart } from 'e2e-utils'
       }
     })
 
+    it('should have shuttle-manifest.json', async () => {
+      const manifest = await next.readJSON(
+        '.next/cache/shuttle/shuttle-manifest.json'
+      )
+      initialGlobalHash = manifest.globalHash
```

Comment:
might be better hygiene to set this in a `beforeAll` just so we don't have weird test failures if this test is ever skipped/moved/removed.